libtgvoip.gyp

```diff
@@ -382,7 +382,7 @@
                 'WEBRTC_POSIX',
               ],
               'conditions': [
-                [ '"<!(uname -p)" == "x86_64" or "<!(uname -p)" == "i686"', {
+                [ '"<!(uname -m)" == "x86_64" or "<!(uname -m)" == "i686"', {
```

Comment:
Don't set SSE2 for x86_64!

Reply:
Ops, sorry... I forgot to remove that... Doing it now.